src/lib/Controls.js

```diff
@@ -35,13 +35,9 @@ class Controls {
     constructor(container) {
         this.containerEl = container;
 
-        const controlsWrapperEl = document.createElement('div');
-        controlsWrapperEl.className = 'bp-controls-wrapper';
-        controlsWrapperEl.setAttribute('data-testid', 'controls-wrapper');
```

Comment:
Make sure this isn't used by any end-to-end tests in other repos (elements, webapp, etc.)

Reply:
> Make sure this isn't used by any end-to-end tests in other repos (elements, webapp, etc.)

Checked (elements, webapp, annotations) and there's no usage of it.